artview/components/select_region.py

```diff
@@ -13,7 +14,7 @@
 
 from ..core import (Variable, Component, common, VariableChoose,
                     componentsList, QtGui, QtCore)
-from ..core.points import Points, write_points_csv, read_points_csv
+from ..core.points import Points
 
```

Comment:
I'm open to that, maybe under "Points Data" or something (whatever is fine). I like the idea of linking to both. 

Let's have parallel SelectRegion (could even rename SelectPoints or something if we need another name) so that we have both ongoing and then narrow down the scope through future commits. What do you think?